core/src/test/java/org/testcontainers/junit/OutputStreamWithTTYTest.java

```diff
@@ -0,0 +1,94 @@
+package org.testcontainers.junit;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import org.junit.Rule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.output.ToStringConsumer;
+import org.testcontainers.containers.output.WaitingConsumer;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Consumer;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.*;
+import static org.testcontainers.containers.output.OutputFrame.OutputType.STDOUT;
+
+public class OutputStreamWithTTYTest {
+
+    @Rule
+    public GenericContainer container = new GenericContainer("alpine:3.2")
+        .withCommand("/bin/sh", "-c", "sleep 2; ls -1" +
```

Comment:
1) could you please remove `+ ""`
2) would be better to not use `sleep` here because it adds constant 2 seconds to the overall test execution time

Reply:
...also, you might want to use `OneShotStartupCheckStrategy` here

Reply:
Sorry, but I was forced to do so because ls command is too fast and container is closes before Testcontainers have time to connect to it.

Solution with Google ping is not the case for me because I behind the firewall and cant ping the Internet. :smile: